Library/Core/UnoCore/Backends/CPlusPlus/Uno/Support.cpp

```diff
@@ -91,6 +97,47 @@ void uFatal(const char* src, const char* msg)
     abort();
 }
 
+uThreadLocal* uCreateThreadLocal(void (*destructor)(void*))
+{
+#ifdef WIN32
+    // TODO: Handle destructor...
```

Comment:
The old implementation in `uno-base` didn't handle this either, meaning we still get a small mem-leak per thread on Win32/C++ builds. I don't think this is a major issue at this point, since it's not a regression, and this backend isn't much used.